graphql/resolvers/RootMutations/claimMembership.js

```diff
@@ -23,11 +24,16 @@ module.exports = async (_, args, {pgdb, req, t}) => {
 
     // commit transaction
     await transaction.transactionCommit()
-
-    return true
   } catch (e) {
     await transaction.transactionRollback()
     logger.info('transaction rollback', { req: req._log(), args, error: e })
     throw e
   }
+
+  await updateUserOnMailchimp({
+    userId: req.user.id,
+    pgdb
+  })
+
```

Comment:
also update the giver – might no longer be a member after giving away all memberships